plugins/proxy/subcommands/set

```diff
@@ -1,14 +1,21 @@
 #!/usr/bin/env bash
 set -eo pipefail; [[ $DOKKU_TRACE ]] && set -x
 source "$PLUGIN_CORE_AVAILABLE_PATH/common/functions"
+source "$PLUGIN_AVAILABLE_PATH/config/functions"
 
 proxy_set_cmd() {
   declare desc="enable proxy for app via command line"
   local cmd="proxy:set"
   [[ -z $2 ]] && dokku_log_fail "Please specify an app to run the command on"
 
   local APP="$2"; verify_app_name "$APP"
-  dokku_log_info1 "proxy:set not implemented"
+  local PROXY_TYPE="$3"
+
+  if [[ -n "$PROXY_TYPE" ]]; then
+    config_set --no-restart "$APP" DOKKU_APP_PROXY_TYPE="$PROXY_TYPE"
```

Comment:
Do we have a list somewhere of "protected" env vars that users should not directly interact with because dokku uses them internally?